<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPlotHistory</class>
 <widget class="QWidget" name="FormPlotHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,9">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="hLay14">
         <item>
          <widget class="QLabel" name="label14">
           <property name="text">
            <string>curve 14bit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnPrev14">
           <property name="text">
            <string>prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnNext14">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelStatus14">
         <property name="text">
          <string>status</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="hLay24">
         <item>
          <widget class="QLabel" name="label24">
           <property name="text">
            <string>curve 24bit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnPrev24">
           <property name="text">
            <string>prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnNext24">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelStatus24">
         <property name="text">
          <string>status</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
